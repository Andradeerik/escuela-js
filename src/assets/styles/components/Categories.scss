.categories__title {
  color: white;
  font-size: 13px;
  position: absolute;
  padding-left: 30px;
}