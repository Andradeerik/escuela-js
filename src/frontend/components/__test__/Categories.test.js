import React from 'react';
import { create } from 'react-test-renderer';
import Categories from '../Categories';

describe('Categories component', () => {
  test('Matches the snapshot', () => {
    const categories = create(<Categories />);
    expect(categories.toJSON()).toMatchSnapshot();
  });
});
