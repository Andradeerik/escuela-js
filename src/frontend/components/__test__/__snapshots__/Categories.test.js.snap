// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories component Matches the snapshot 1`] = `
<div
  className="Categories"
>
  <h3
    className="categories__title"
  />
</div>
`;
